services/chaindb/postgresql/validatorepochsummaries.go

```diff
@@ -518,7 +518,7 @@ WHERE f_validator_index = $1
 }
 
 // PruneValidatorEpochSummaries prunes validator epoch summaries up to (but not including) the given point.
-func (s *Service) PruneValidatorEpochSummaries(ctx context.Context, to phase0.Epoch, retain []phase0.ValidatorIndex) error {
+func (s *Service) PruneValidatorEpochSummaries(ctx context.Context, to phase0.Epoch, retainPubkeys []phase0.BLSPubKey) error {
```

Comment:
I think we can keep this value as `retain` given that its type is `BLSPubKey`

Reply:
Updated !